scripts/install_third_party.py

```diff
@@ -15,12 +15,14 @@
 """Installation script for Oppia third-party libraries."""
 
 import itertools
+import json
 import os
 import shutil
 import StringIO
 import tarfile
 import urllib
 import urllib2
+import utils
```

Comment:
Don't import 'utils' -- that's an oppia-specific file. That's why Travis is having issues. Instead, just replicate the logic in this file; it's a very short method.